MultipleTopLevelWebVIs/Readme.md

```diff
@@ -3,74 +3,65 @@
-1. Open `C:\Program Files\National Instruments\Shared\Web Server\htdocs`
-2. Copy the `MultipleTopLevelWebVIs` directory into the `htdocs` directory
-3. Open a Web Browser and navigate to `http://localhost/MultipleTopLevelWebVIs/index.html`
+1. Clone the [ni/webvi-examples](https://github.com/ni/webvi-examples) repository to your machine.
+2. Open `Multiple Top-Level VIs\Multiple Top-Level VIs.lvproject`.
+3. Open `Main.gviweb` and click the **Run** button.  
+**Note:** Clicking a link to another top-level WebVI within LabVIEW opens a web browser, but the WebVI does not run in the web browser. <!-- TODO DE12694: Pressing run and clicking link to other top-level panel opens a link to broken WebVI stuck in synchronization mode -->
+4. Test the top-level WebVIs.  
+  a. On the **Project Files** tab, double-click **System Designer** to open it.  
+  b. On the **Web Server** target, right-click `MultipleTopLevelVIs.gcomp` and select **Run Web Component**.  
+  **Note:** Clicking **Run Web Component** opens all the top-level WebVIs in your default browser at once.
+  <!-- TODO DE12779: Performing Run Web Component opens all Top-Level Vis at the same time -->
+5. Build the web application.  
+  a. Open `WebApp.gcomp`.  
+  b. On the **Document** tab, click **Build**.
+
+# Hosting a Web Application on a Server
+Copy and paste the entire `MultipleTopLevelVIs` directory built under `<Project Directory>/Builds/Web Server/Configuration1/` to any web server of your choosing.
+
+## Hosting on the LabVIEW 2009-2017 Web Server
+1. Navigate to `<National Instruments>\Shared\NI WebServer\www`.
+2. Copy the `MultipleTopLevelVIs` directory into the `www` directory.
+3. Open a web browser and navigate to `http://localhost:8080/MultipleTopLevelVIs/index.html`.
+
+## Hosting on the NI SystemLink Server
+1. Navigate to `<National Instruments>\Shared\Web Server\htdocs`.
+2. Copy the `MultipleTopLevelVIs` directory into the `htdocs` directory.
+3. Open a web browser and navigate to `http://localhost/MultipleTopLevelVIs/index.html`.
 
 # Details
 
 ## File Name and Relative URL
-Each Top-Level WebVI will create a corresponding HTML page where the organization of those pages is determined by position of the WebVI in the Component.
-As can be seen in the following screenshot you have the ability to change the generated File Name for each WebVI as well as a seeing the Relative URL used for linking.
+Each of the top-level WebVIs generates a corresponding HTML file. The organization of the generated HTML files is determined by the order of the top-level WebVIs in the `WebApp.gcomp` file.
 
-![Main.gviweb Right Rail View in Component showing File Name and Relative URL properties](ComponentRightRail.png)
+As you can see in the following screenshot, you can change the file name for each WebVI and view the relative URL used for linking on the **Item** tab.
+![Main.gviweb Item tab in .gcomp file showing File name and Relative URL fields](ComponentRightRail.png)
```

Comment:
Maybe add a newline before the image to create some space between the previous line and the image?